src/sentry/integrations/vsts/integration.py

```diff
@@ -356,19 +359,19 @@ class VstsIntegrationProvider(IntegrationProvider):  # type: ignore
     def post_install(
         self,
         integration: IntegrationModel,
-        organization: Organization,
+        organization: RpcOrganizationSummary,
         extra: Mapping[str, Any] | None = None,
     ) -> None:
-        repo_ids = Repository.objects.filter(
+        repos = repository_service.get_repositories(
             organization_id=organization.id,
-            provider__in=["visualstudio", "integrations:vsts"],
-            integration_id__isnull=True,
-        ).values_list("id", flat=True)
+            providers=["visualstudio", "integrations:vsts"],
+            has_integration=False,
+        )
 
-        for repo_id in repo_ids:
+        for repo in repos:
             migrate_repo.apply_async(
```

Comment:
We might want to make this into an RPC/outbox message as well, similar to other repo migrations.